projects.go

```diff
@@ -464,59 +465,85 @@ func (s *ProjectsService) GetProjectEvents(pid interface{}, opt *GetProjectEvent
 	return p, resp, err
 }
 
+// ContainerExpirationPolicy represents the container expiration attributes for a project.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/projects.html
+type ContainerExpirationPolicy struct {
+	Cadence         string     `json:"cadence"`
+	KeepN           int        `json:"keep_n"`
+	OlderThan       string     `json:"older_than"`
+	NameRegexDelete string     `json:"name_regex_delete"`
+	NameRegexKeep   string     `json:"name_regex_keep"`
+	Enabled         bool       `json:"enabled"`
+	NextRunAt       *time.Time `json:"next_run_at"`
+}
+
+// ContainerExpirationPolicyAttributes represents the container expiration attributes for a project.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/projects.html
+type ContainerExpirationPolicyAttributes struct {
```

Comment:
I think you missed this one:
```suggestion
type ContainerExpirationPolicyOptions struct {
```